GeneralsMD/CMakeLists.txt

```diff
@@ -12,5 +12,17 @@ set(CMAKE_RUNTIME_OUTPUT_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR})
 # Add main build targets
 add_subdirectory(Code)
 
+# If we are building on windows for windows, try and get the game install path from the registry.
+if(CMAKE_HOST_SYSTEM_NAME STREQUAL "Windows" AND CMAKE_SYSTEM_NAME STREQUAL "Windows")
+    get_filename_component(ZEROHOUR_INSTALL_PREFIX  "[HKEY_LOCAL_MACHINE\\SOFTWARE\\Electronic Arts\\EA Games\\Command and Conquer Generals Zero Hour;InstallPath]" ABSOLUTE CACHE)
+endif()
+
+if(ZEROHOUR_INSTALL_PREFIX AND NOT ZEROHOUR_INSTALL_PREFIX STREQUAL "/registry")
+    install(TARGETS z_generals RUNTIME DESTINATION "${ZEROHOUR_INSTALL_PREFIX}")
```

Comment:
Is the `OPTIONAL` absence intended?

Reply:
Game binary is generated no matter what at the moment right?

Reply:
Yes, unless GENZH_BUILD_ZEROHOUR is disabled.

Reply:
In which case nothing in the CMakeLists.txt for that target will get run, including the install setup code.